lib/shelter.rb

```diff
@@ -1,2 +1,34 @@
+require 'pry'
 class Shelter
+    attr_reader :name, :capacity, :pets, :call_pets
+
+    def initialize(name, capacity)
+        @name = name
+        @capacity = capacity
+        @pets = []
+        @call_pets = []
+
+    end
+
+    def add_pet(pet)
+        @pets.push(pet)
+       
+    end
+
+    def call_pets
+        @pets.each { |pets| pets.concat('!') }
+
+        #@call_pets << pets
+    end
+
+    def over_capacity?
```

Comment:
Good job with this method!  Another way you could do the same thing is by relying on the return value of the condition.  In ruby, a condition will always evaluate to either true or false.  

```
def over_capacity?
  @pets.length > @capacity
end
```